metric.go

```diff
@@ -32,12 +33,28 @@ type PartitionedMetric interface {
 
 //go:generate msgp
 
+type MetricId [16]byte
```

Comment:
Wouldn't it make sense to make this type identical with the `MetricID` here and then just embed it into `MetricData`: https://github.com/grafana/metrictank/blob/master/idx/idx.go#L38

Then we could directly use it as it is when ingesting into the in-memory index.

Reply:
IIRC our long term plan was to replace the keys in `MemoryIdx.DefById` with a numeric type, right? If this `schema.MetricId` would have the org id like `idx.MetricID` does, then we could use this type as key for `DefById` in the future.